test/flows/business_coronavirus_support_finder_flow_test.rb

```diff
@@ -0,0 +1,255 @@
+        end
+      end
+    end
+  end
+
+  context "question: self_employed?" do
+    setup do
+      testing_node :self_employed?
+      add_responses business_based?: "england",
+                    business_size?: "0_to_249",
+                    paye_scheme?: "yes"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      %w[yes no].each do |response|
+        should "have a next node of non_domestic_property? for a #{response} response" do
+          assert_next_node :non_domestic_property?, for_response: response
+        end
+      end
+    end
+  end
+
+  context "question: non_domestic_property?" do
+    setup do
+      testing_node :non_domestic_property?
+      add_responses business_based?: "england",
+                    business_size?: "0_to_249",
+                    paye_scheme?: "yes",
+                    self_employed?: "yes"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      %w[yes no not_sure].each do |response|
+        should "have a next node of sectors? for a #{response} response" do
+          assert_next_node :sectors?, for_response: response
+        end
+      end
+    end
+  end
+
+  context "question: sectors?" do
+    setup do
+      testing_node :sectors?
+      add_responses business_based?: "england",
+                    business_size?: "0_to_249",
+                    paye_scheme?: "yes",
+                    self_employed?: "yes",
+                    non_domestic_property?: "yes"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of closed_by_restrictions? for any sector" do
+        responses = %w[
+          nurseries
+          retail_hospitality_or_leisure
+          nightclubs_or_adult_entertainment
+          personal_care
+        ]
+
+        assert_next_node :closed_by_restrictions?, for_response: responses
+      end
+
+      should "have a next node of closed_by_restrictions? for no sectors" do
+        assert_next_node :closed_by_restrictions?, for_response: %w[none]
+      end
+    end
+  end
+
+  context "question: closed_by_restrictions?" do
+    setup do
+      testing_node :closed_by_restrictions?
+      add_responses business_based?: "england",
+                    business_size?: "0_to_249",
+                    paye_scheme?: "yes",
+                    self_employed?: "yes",
+                    non_domestic_property?: "yes",
+                    sectors?: %w[none]
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of results for any response" do
+        assert_next_node :results, for_response: %w[local_1 local_2 national]
+      end
+
+      should "have a next node of results for not closed_by_restrictions" do
```

Comment:
I don't think this test is needed as the condition of "have a next node of results for any response" in the previous test covers this - and there is no additional code for this.